-c option causes panic when some key is omitted

Hello!
If the configuration file does not contain ltsv, json, regexp, or pcap, it will cause `nil pointer dereference` error

```
❯ ./cli/alp/alp json -c 'example/config.yml'
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x0 pc=0x1006cd7d0]

goroutine 1 [running]:
github.com/tkuchiki/alp/options.LoadOptionsFromReader(0x10089af28, 0x14000010bd8, 0x0, 0x0, 0x14000010bd8)
	/Users/hosshii/Workspace/github.com/tkuchiki/alp/options/option.go:583 +0x600
github.com/tkuchiki/alp.(*Profiler).Run(0x14000139780, 0x14000020090, 0x3, 0x3, 0x0, 0x0)
	/Users/hosshii/Workspace/github.com/tkuchiki/alp/alp.go:138 +0x730
main.main()
	/Users/hosshii/Workspace/github.com/tkuchiki/alp/cli/alp/main.go:13 +0x9c
```

I think this is because [this field](https://github.com/tkuchiki/alp/blob/3dd69adb9e69dfbf4a59f169a341ffc886fa9d82/options/option.go#L79) is pointer type, and the field become `nil` if configuration is omitted.

--- a/options/option.go
+++ b/options/option.go
@@ -533,7 +533,7 @@ func LoadOptionsFromReader(r io.Reader) (*Options, error) {
 		return opts, err
 	}
 
-	configs := &Options{}
+	configs := NewOptions()
 	err = yaml.Unmarshal(buf, configs)
 
 	opts = SetOptions(opts,

